### PR TITLE
Refactor so de-integration can be used elsewhere in CocoaPods

### PR DESCRIPTION
**Pull request by segiddins**
Should also manage to close https://github.com/CocoaPods/cocoapods-deintegrate/issues/10.

\c @kylef

### CODE REVIEW

File: spec/fixtures/Project/None/TestProject.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -405,6 +405,7 @@
 				276A2C831A7C1683004BCC6F /* Release */,
 			);
 			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
```
Comment (kylef) [inline spec/fixtures/Project/None/TestProject.xcodeproj/project.pbxproj]: Hmm, I was weary about this. It's not actually put into projects made via Xcode by default. Only Xcodeproj adds it when you save the project via Xcodeproj.

**Comment by kylef** [comment]
:+1:

File: spec/fixtures/Project/None/TestProject.xcodeproj/project.pbxproj (hunk 1 above)
Comment (segiddins) [inline spec/fixtures/Project/None/TestProject.xcodeproj/project.pbxproj]: Actually, Xcode made this change for me when I opened the project

Comment (kylef) [inline spec/fixtures/Project/None/TestProject.xcodeproj/project.pbxproj]: Oh then :+1:, it didn't do this when I first created the fixtures.

**Pull request merged**